src/test/scala/org/broadinstitute/hail/methods/SkatoSuite.scala

```diff
@@ -0,0 +1,387 @@
+package org.broadinstitute.hail.methods
+
+import org.broadinstitute.hail.Utils._
+import org.broadinstitute.hail.annotations.Annotation
+import org.broadinstitute.hail.check.Prop._
+import org.broadinstitute.hail.check._
+import org.broadinstitute.hail.driver._
+import org.broadinstitute.hail.expr._
+import org.broadinstitute.hail.variant._
+import org.broadinstitute.hail.{FatalException, SparkSuite}
+import org.testng.annotations.Test
+
+import scala.collection.mutable
+import scala.language._
+import scala.math.Numeric.Implicits._
+import scala.sys.process._
+
+
+class SkatoSuite extends SparkSuite {
+
+  val testVepSignature = TStruct("transcript_consequences" -> TArray(TStruct("lof_flag" -> TBoolean, "gene_id" -> TString)))
+  val sampleSignature = TStruct("phenotype" -> TBoolean)
+
+  def writeConfig(file: String) = writeTextFile(file, sc.hadoopConfiguration) { w =>
+    w.write("hail.skato.Rscript Rscript\n")
+    w.write("hail.skato.script src/dist/scripts/skato.r\n")
```

Comment:
If these are the defaults this isn't needed.  If you want to keep it, just add it to test resources.